cfy_manager/main.py

```diff
@@ -527,6 +529,32 @@ def _create_component_objects():
         )
 
 
+def _delete_patterns_from_file(path, patterns_list):
+    try:
+        for pattern in patterns_list:
+            for line in (fileinput.input(path, inplace=1)):
+                sys.stdout.write(line.replace(pattern, ""))
+    except OSError:
+        logger.error("The file `{0}` could not be found".format(path))
+
+
+def _remove_rabbitmq_service_unit():
+    prefix = "/lib/systemd/system"
+    basic_pattern = "cloudify-rabbitmq.service"
+    services = ["cloudify-amqp-postgres.service",
```

Comment:
Why not put all the services in the list as a tuple.
That way each service has its pattern it needs to check in a single element in the list.
Something like:
```
services_and_patterns = \
    [("cloudify-amqp-postgres.service", [basic_pattern]),
     ("cloudify-mgmtworker.service", ["Wants={0}".format(basic_pattern), "After={0}".format(basic_pattern)])]
for service, pattern_list in services_and_patterns:
    for pattern in pattern_list:
        replace_in_file(....)```
etc etc...